src/main/java/org/cascadebot/cascadebot/music/CascadePlayer.java

```diff
@@ -135,7 +134,7 @@ public boolean isShuffleEnabled() {
 
     public boolean skip() {
         try {
-            player.playTrack(tracks.remove());
+            player.stopTrack();
```

Comment:
?

Reply:
The old method broke things

Reply:
But this will no longer queue the next track?

Reply:
It does, cause it trigger the track end event

Reply:
Gotcha